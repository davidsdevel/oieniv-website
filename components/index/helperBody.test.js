/*eslint-env jest*/
import React from "react";
import HelperBody from "./helperBody";
import {shallow} from "enzyme";
import renderer from "react-test-renderer";

describe("HelperBody with Enzyme", () => {
	it("Render HelperBody without crash", () => {
		const wrapper = shallow(<HelperBody name="Pronani" content="Hola Mundo" logoURL="/static/images/test-logo.png"/>);
	
		expect(wrapper.find("p").text()).toEqual("Hola Mundo");
		expect(wrapper.find("h4").text()).toEqual("Pronani");
		expect(wrapper.find("img").prop("src")).toEqual("/static/images/test-logo.png");
	});
});

describe("HelperBody with Snapshot", () => {
	it("Render HelperBody without crash", () => {
		const component = renderer.create(<HelperBody name="Pronani" content="Hola Mundo" logoURL="/static/images/test-logo.png"/>);
		const tree = component.toJSON();

		expect(tree).toMatchSnapshot();
	});
});
