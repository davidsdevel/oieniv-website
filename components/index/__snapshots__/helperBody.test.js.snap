// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`HelperBody with Snapshot Render HelperBody without crash 1`] = `
<div
  className="helper-body"
>
  <h4>
    Pronani
  </h4>
  <img
    alt="Organos Auxiliares - Pronani - OIENIV"
    src="/static/images/test-logo.png"
  />
  <p>
    Hola Mundo
  </p>
</div>
`;
